fpdf/fpdf.py

```diff
@@ -2392,21 +2391,24 @@ def text(self, x, y, txt=""):
             txt2 = escape_parens(txt_mapped.encode("utf-16-be").decode("latin-1"))
         else:
             txt2 = escape_parens(txt)
-        s = f"BT {x * self.k:.2f} {(self.h - y) * self.k:.2f} Td"
+        sl = [f"BT {x * self.k:.2f} {(self.h - y) * self.k:.2f} Td"]
```

Comment:
Maybe a more explicit variable name would be welcome?

Reply:
"sl" = "string list", the same as in _render_styled_text_lines(). I think its purpose is quite obvious... :wink: